eng/pipelines/dotnet-sqlclient-ci-core.yml

```diff
@@ -41,6 +41,18 @@ parameters:
   type: object
   default: [false, true]
 
+- name: SNIVersion
+  displayName: |
+    Use a specific SNI version from the internal validation feed
```

Comment:
As a display name, I'm thinking of how this would be displayed in the new run dialog. This is really long, and could be reduced to "SNI NuGet Version Override"